CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 10.0.3
+
+* Add creditCardExpirationDate
+* Add creditCardExpirationDateNotExpired
+* Add creditCardCVC
+* Add colorCode
+* Add phoneNumber
+* Add uppercase
+* Add lowercase
+* Add file extension
+* Add max file size
+* Add notMatch
+* Add range
+* Add date range
+* Add must be true
+* Add must be false
+* Add special characters
+* Add numeric characters
+* Add lowercase characters
+* Add uppercase characters
+* Add password
```

Comment:
We don't generate 10.0.2 version, so can unify the 10.0.2 version change and this list, on a new that would be 10.1.0, because is a several improve changes.